src/config/product.openshiftMetrics.js

```diff
@@ -243,7 +244,7 @@ const config = {
               isInline
               component="a"
               variant="link"
-              href={`${helpers.UI_DEPLOY_PATH_LINK_PREFIX}/openshift/details/${instanceId}`}
+              href={`${helpers.UI_DEPLOY_PATH_LINK_PREFIX}/openshift/details/${inventoryId}`}
```

Comment:
question:
The SWATCH issue specifically calls out RHEL. You confirmed `OpenShift Dedicated` and `RHACS` in your PR/MR notes, but not `OpenShift Metrics`... is this correct and the notes just skipped it?

Reply:
I've just confirmed that the OpenShift Metrics should also use the inventory ID. Description updated.